mmv1/templates/terraform/examples/cloudbuild_trigger_allow_exit_codes.tf.erb

```diff
@@ -0,0 +1,66 @@
+resource "google_cloudbuild_trigger" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
This  example needs to be added to the file mmv1/products/cloudbuild/Trigger.yaml.

This is an example https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/cloudbuild/Trigger.yaml#L39